plugins/inventory/linode.py

```diff
@@ -79,6 +87,13 @@
   mailservers: "'mail' in (tags|list)"
 compose:
   ansible_port: 2222
+
+# Example where control traffic limited to internal network
+plugin: community.general.linode
+access_token: foobar
+ip_style: api
+compose:
+  ansible_host: "ipv4 | json_query('[?public==`false`].address') | first"
```

Comment:
```suggestion
  ansible_host: "ipv4 | community.general.json_query('[?public==`false`].address') | first"
```

Reply:
I will apply this @felixfontein , but for what it's worth, the `scope.function` syntax does not work for me locally even though the documentation says that it should. I am using Ansible as installed from the Fedora 34 repositories.